File: python/485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        length = 0
        temp = 0
        for i in nums:
            if i == 1:
                temp += 1
            else:
                if temp > length :
                    length =  temp
                temp = 0
        if temp > length :
            length =  temp
        return length

File: java/485.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int ans = 0;
        int tmp = 0;
        for(int i = 0; i< nums.length;i++){
            if(nums[i] == 1){
                tmp ++;
                ans = Math.max(tmp,ans);
            }else{
                tmp = 0;
            }
        }
        return ans;
        
    }
}